### ISSUE TITLE
Temporarily disable pull to refresh

### ISSUE THREAD

**Issue by repetix** [aagarwal1012/Liquid-Pull-To-Refresh#70]
I have a situation where I need to temporarily disable the pull to refresh feature. Is there any way to do this programmatically with state (using GetX for my project so it is stateless and I would use obx observable for this). How would you suggest I do this without forking your project?  Thanks

**Comment by repetix** [comment]
Please respond to this! Before, it was just an issue with timing and workflow, but now (since Flutter 3x) the pull to refresh is no longer compatible with ReorderableListView. I can no longer implement both reordering and pull to refresh on the same page, even when using a custom version of ReorderableDragStartListener (which used to work, albeit not as well as it should have). I love your package, but I have to allow customers to reorder items more than refreshing them, so I have to remove the capability until this is resolved. Please let me know when you can get to this or if you have any suggestions on how I can code a workaround.

**Comment by MHamzaAhmad** [comment]
@aagarwal1012 Can you please assign this to me? I want to work on this.

**Comment by aagarwal1012** [comment]
@MHamzaAhmad please go ahead.

**Comment by HassanTeslim007** [comment]
@MHamzaAhmad did you make any breakthrough with this issue?

**Comment by MHamzaAhmad** [comment]
Yeah, i did made something but that was a while ago, like a year maybe. I don’t have the code for that right now as I couldn’t push it without permissions.Regards,HamzaOn 09-Oct-2024, at 2:24 PM, Alpha ***@***.***> wrote:﻿
@MHamzaAhmad did you make any breakthrough with this issue?
